spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -460,6 +460,13 @@ create table Book (
 
 NOTE: Support for entity associations is not required by this specification.
 
+==== Entity Names
+
+Each entity must be assigned an _entity name_ by the provider.
+By default, this must be the unqualified Java class name of the entity class.
```

Comment:
Good question: I would like to say "no", and it's the user responsibility to make sure entity names are unique. It's annoying that the facility to do that would by external to the spec.

Reply:
It's actually _really_ hard to write down something meaningful here, since the JD spec doesn't have any well-defined notion of a "persistence unit", that is, of which entities are eligible for use in query methods of a given repository.

Reply:
If someone thinks we need to say something specific about the issue of name collisions, please propose some language. 

I think it's acceptable if the spec stays silent on this issue for now. If it's _not_ acceptable, I think we can fix the problem in a separate discussion/PR. It doesn't need to block the merge of this PR.